vulnerabilities/importers/apache_httpd.py

```diff
@@ -49,47 +62,110 @@ def updated_advisories(self):
         # skips processing the response further to avoid duplicate work
 
         if create_etag(data_src=self, url=self.url, etag_key="ETag"):
```

Comment:
IMHO this won't work becuase `self.url == "https://httpd.apache.org/security/json/{}"`

Reply:
This won't be an issue if we go with `base_url` approach.